docs/integrations/creating-nodes/build/n8n-node.md

```diff
@@ -0,0 +1,180 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+contentType: howto
+---
+
+# Using the n8n-node tool
+
+The `n8n-node` tool is the official CLI for developing community nodes for n8n. You can use it to scaffold out new nodes, build your projects, and run your node as you develop it.
+
+Using `n8n-node`, you can create nodes that adhere to the 
+
+## Prerequisites
+
+Before you can use the `n8n-node` tool, you must [install n8n locally with `npm`](/hosting/installation/npm.md). The `n8n-node` tool runs the local n8n instance with your community node loaded to provide live previews.
+
+## Get n8n-node
+
+### Run n8n-node without installing
+
+You can run `n8n-node` directly without installing with `npx`.
+
+For example, to run the `n8n-node new` command, you can type:
+
+```shell
+npx n8n-node new
```

Comment:
This doesn't work.

```
> npx n8n-node new
npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/n8n-node - Not found
npm error 404
npm error 404  'n8n-node@*' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.
npm error A complete log of this run can be found in: /Users/jon/.npm/_logs/2025-08-27T08_27_01_734Z-debug-0.log
```

Using `npx @n8n/node-cli` seems to be the better option

Reply:
Yeah, I tried to mention this in the PR description.

This command is in the README, so I assumed it's supposed to eventually work. The behavior today is actually better than what it was a few days ago, when even this wasn't working. As you said though, the best way currently with `npx` seems to be either `npx @n8n/node-cli` or `npx --package=@n8n/node-cli n8n-node` depending on whether we think keeping the actual command name the same is less confusing for users.

Reply:
I suspect it was only published on Monday so what we see now is how it will be for users. I wouldn't be against dropping the npx part fully and just focusing on installing the package

Reply:
Okay, thanks for the added context. I think I'm just going to fix it up so that it's functional for now, but if it seems like it might be causing confusion when we get this in front of users, I'm not against removing it.

Reply:
Okay, should run as written now at least.